src/test/java/org/sagebionetworks/bridge/models/assessments/ImageResourceTest.java

```diff
@@ -0,0 +1,48 @@
+package org.sagebionetworks.bridge.models.assessments;
+
+import static org.sagebionetworks.bridge.TestConstants.LABELS;
+
+import static org.testng.Assert.assertEquals;
+
+import org.sagebionetworks.bridge.json.BridgeObjectMapper;
+import org.testng.annotations.Test;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class ImageResourceTest {
+    @Test
+    public void canSerialize() throws JsonMappingException, JsonProcessingException {
+        ImageResource imageResource = new ImageResource();
+        imageResource.setName("default");
+        imageResource.setModule("sage_survey");
+        imageResource.setLabels(LABELS);
+
+        JsonNode node = BridgeObjectMapper.get().valueToTree(imageResource);
+
+        assertEquals(node.get("name").textValue(), "default");
```

Comment:
Optional addition: assert the node size. It would cause the test to fail if someone adds another field but forgets to update the unit tests.